package.json

```diff
@@ -27,6 +27,7 @@
   },
   "dependencies": {
     "@altinn/figma-design-tokens": "^0.3.0",
+    "@radix-ui/react-popover": "^1.0.0",
```

Comment:
This needs to be added to the `exclude` list in Rollup config. We're going to [automate this](https://github.com/Altinn/altinn-design-system/issues/62) , but until then it needs to be added manually